test/kane_test.exs

```diff
@@ -1,4 +1,22 @@
 defmodule KaneTest do
   use ExUnit.Case
   doctest Kane
+
+  defmodule GothConfigMock do
+    def get(account, :project_id) do
+      {:ok, account}
+    end
+  end
+
+  describe "project/0" do
+    test "with no account configured" do
+      assert :default == Kane.project(GothConfigMock)
+    end
+
+    test "with an account configured" do
+      Application.put_env(:kane, :account, "test@account.com")
+      assert "test@account.com" == Kane.project(GothConfigMock)
+      Application.delete_env(:kane, :account)
```

Comment:
Instead of deleting it would be nice to set it to the previous value.

Reply:
In this case there is no previous value, the key is not present. Setting it to `nil` would not be equivalent either.

Reply:
The config is shared across all tests, another test might set it up.

Reply:
Right, what I meant was that if the config key is not set, there is no _previous value_. And if we use `nil` we would end up with something different (no key vs key with value nil).

But I see your point, I've changed this to keep the previous 👍🏻